<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuForm</class>
 <widget class="QMainWindow" name="MenuForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Меню</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="centralwidgetLayout">
    <item row="1" column="2">
     <widget class="QWidget" name="mainWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>700</width>
        <height>700</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="mainWidgetLayout">
       <property name="leftMargin">
        <number>75</number>
       </property>
       <property name="topMargin">
        <number>75</number>
       </property>
       <property name="rightMargin">
        <number>75</number>
       </property>
       <property name="bottomMargin">
        <number>75</number>
       </property>
       <item>
        <widget class="QPushButton" name="libraryBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Каталог книг библиотеки</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="userBooksBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Ваши книги</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="logOutBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Выход</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
